BenchmarkVCFs/ExportToGoogleSheets.ipynb

```diff
@@ -0,0 +1,315 @@
+        "\n",
+        "__*Note*__: The Google authentication process requires this notebook to be run in [Google Colab](https://colab.research.google.com/) using the link above."
+      ],
+      "metadata": {
+        "id": "wuuQdfr022Nw"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Install the Firecloud API\n",
+        "\n",
+        "This is required in case the [Firecloud API](https://github.com/broadinstitute/fiss) isn't installed in the current runtime."
+      ],
+      "metadata": {
+        "id": "Zg0RWhvyXgQH"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "try:\n",
+        "    import firecloud.api as fapi\n",
+        "except ModuleNotFoundError:\n",
+        "    %pip install firecloud"
+      ],
+      "metadata": {
+        "id": "Wp4aXPPt4vtX"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Data export\n",
+        "\n",
+        "In the cell below, enter the `gs://` path to the `comparison.csv` file, which is the output of the `CompareBenchmarks` workflow.\n",
+        "\n",
+        "In addition, please enter the Google Spreadsheet ID of the notebook that you want to export the data to. The spreadsheet ID is shown in the URL in the browser. If you want to create a new spreadsheet you can do so on [sheets.google.com](https://sheets.google.com) or simply navigate to [sheet.new](https://sheet.new).\n",
+        "\n",
+        "The notebook will automatically create a new sheet (tab) in the provided spreadsheet, named with the current date and time (in UTC).\n",
+        "\n",
+        "__*Note*__: When executing the cell you will be provided with a link to authenticate this notebook to access your Google sheets. Please click that link, authenticate with Google, and enter the resulting code into the prompt in this notebook."
+      ],
+      "metadata": {
+        "id": "xrqR2vZWa_M9"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "comparison_csv_gs_path = \"gs://fc-640bf3c9-3bb5-45c1-a59f-a271111337da/683fd921-ab36-4382-8aa7-eb5d27716469/CompareBenchmarks/4fff91a6-e2d1-442f-869c-92134b84d092/call-CompareBenchmarksTask/output.csv\" #@param {type:\"string\"}\n",
+        "\n",
+        "spreadsheet_id = \"1kVu86hxxdsHe7bxXpKq5cIcPA29v8rQl9RU5TEss8-Q\" #@param {type:\"string\"}\n",
+        "\n",
+        "import firecloud.api as fapi\n",
+        "import datetime\n",
+        "from google.colab import auth\n",
+        "auth.authenticate_user()\n",
+        "from googleapiclient.discovery import build\n",
+        "\n",
+        "service = build('sheets', 'v4')\n",
+        "\n",
+        "csv_lines = !gsutil cat {comparison_csv_gs_path}\n",
```

Comment:
I imagine you can use the gcs api to read from the blob instead of calling gsutil?

Reply:
Done